subsys/caf/modules/sensor_manager.c

```diff
@@ -490,7 +494,12 @@ static bool handle_wake_up_event(const struct app_event_header *aeh)
 			int ret = 0;
 
 			if (sc->trigger) {
-				sensor_trigger_set(sc->dev, &sc->trigger->cfg, NULL);
+				ret = sensor_trigger_set(sc->dev, &sc->trigger->cfg, NULL);
+				if (ret < 0) {
+					LOG_ERR("Cannot activate a sensor %s trigger (%d)",
+						sc->dev->name, ret);
+					ret = 0;
```

Comment:
Why this line?

Reply:
ret can be not changed and will change behavior of this if:
 `if (!ret) {
	LOG_DBG("Sensor %s wake up", sc->dev->name);
	sensor_wake_up_post(sc, sd);
	}`